src/clj/rems/api/applications.clj

```diff
@@ -206,7 +209,20 @@
       :summary "Get current user's all applications"
       :roles #{:logged-in}
       :return GetApplicationsResponse
-      (ok (applications/get-user-applications (getx-user-id))))
+      ;; XXX: the old API doesn't know about members, so it doesn't return applications where the user is a member
+      ;; XXX: this code cannot be moved to rems.db.applications because it would create a cyclic dependency
```

Comment:
What cylic dependency? `rems.api.applications` depends on `rems.db.applications` but there shall be no other direction dependency.

Reply:
There are actually two cycles that would be produced by both `rems.api.applications/api-get-application` and `rems.db.dynamic-roles/permissions-of-all-applications`.

I extracted `rems.api.applications/get-user-applications` so at least it isn't inlined inside the router.